### ISSUE TITLE
1.35.0 panics on `completions`

### ISSUE THREAD

**Issue by Ugzuzg** [Schniz/fnm#1006]
Steps to reproduce:
1. Download `fnm-linux.zip` from the release: https://github.com/Schniz/fnm/releases/tag/v1.35.0
2. Execute `/fnm completions`

```
$ ./fnm completions --shell zsh
thread 'main' panicked at 'built', /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/clap_complete-4.3.1/src/generator/utils.rs:124:26
```

**Comment by es20490446e** [comment]
https://github.com/rust-lang/rust/commit/08ce68b6a6bad360e9c3611ad60cf6598401f878
https://github.com/rust-lang/rust/commit/05bc71cfaf9eb35caab425126d4fa8dcb8c62072

**Comment by salimp2009** [comment]
> [rust-lang/rust@08ce68b](https://github.com/rust-lang/rust/commit/08ce68b6a6bad360e9c3611ad60cf6598401f878) [rust-lang/rust@05bc71c](https://github.com/rust-lang/rust/commit/05bc71cfaf9eb35caab425126d4fa8dcb8c62072)

if it is fixed with those commits, it is not working in 1.35.0 release.
Also when installed with Cargo it installs version 1.33 ?

**Comment by es20490446e** [comment]
v1.33.1

**Comment by salimp2009** [comment]
nice. looks like it is fixed. Hope this gets into the 1.35 release

**Comment by salimp2009** [comment]
when do you think you can fix this for 1.35 is still broken as it is now unless those new commits are not   merged into 1.35 files as well. right ?
